sdk/loadtesting/azure-developer-loadtesting/tests/testcase.py

```diff
@@ -38,4 +41,7 @@ def create_run_client(self, endpoint):
     loadtesting_app_component="000",
     loadtesting_subscription_id="000",
     loadtesting_resource_id="000",
+    loadtesting_test_profile_id="000",
+    loadtesting_target_resource_id="000",
```

Comment:
For `loadtesting_endpoint`, we will change it to `"00000000-0000-0000-0000-000000000000.eastus.cnt-prod.loadtesting.azure.com`

For `loadtesting_target_resource_id`, we will change it to `"/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myRG/providers/Microsoft.Web/sites/myFlexFunction"`